integreat_cms/cms/views/pages/page_actions.py

```diff
@@ -896,3 +897,62 @@ def refresh_date(
             "region_slug": request.region.slug,
         },
     )
+
+
+@require_POST
+@json_response
+# pylint: disable=unused-argument
+def post_hix_and_word_number_per_page(request, region_slug):
+    """
+    This function collects the hix score and the amount of words per page from the source
+
+    :param request: The current request
+    :type request: ~django.http.HttpRequest
+
+    :param region_slug: The slug of the current region
+    :type region_slug: str
+
+    :return: A json response of {{"id": id, "title": title, "words": number of words, "hix": if HIX value is high enough}} in case of success
+    :rtype: ~django.http.JsonResponse
+    """
+    selected_page_ids = json.loads(request.body.decode("utf-8"))
+    selected_pages = request.region.get_pages(return_unrestricted_queryset=True).filter(
+        id__in=selected_page_ids
+    )
+    hix_threshold = settings.HIX_REQUIRED_FOR_MT
+
+    filtered_pages = {}
+    translatable = {}
+    not_translatable = {}
+
+    for page in selected_pages:
+        translation = page.get_translation(request.region.default_language.slug)
```

Comment:
In my opinion, we should just skip the HIX value check in this case. I would be surprised if the HIX value of the German source text had a huge influence on the quality of machine translations from and to completely different languages.
Also, this would reflect the same constraints that we enforce in the forms (where you can enable machine translations for any language other than German independently from the HIX score).